reference/5.1/Microsoft.PowerShell.Utility/Write-Output.md

```diff
@@ -150,6 +150,10 @@ You can pipe objects to `Write-Output`.
 
 ## RELATED LINKS
 
+[about_Output_Streams](/powershell/module/Microsoft.PowerShell.Core/About/about_Output_Streams.md)
+
+[about_Redirection](/powershell/module/Microsoft.PowerShell.Core/About/about_Redirection.md)
```

Comment:
**Site Relative** links shouldn't include the extension.

Reply:
fixed